src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2026,16 +2026,15 @@ Json::Value NetworkOPsImp::getServerInfo (bool human, bool admin)
             if (std::abs (closeOffset.count()) >= 60)
                 l[jss::close_time_offset] = closeOffset.count();
 
-            // VFALCO How do we fix this?
-            /*
-            std::uint32_t lCloseTime (lpClosed->info().closeTime);
+            std::uint32_t lCloseTime = lpClosed->info().closeTime;
+            std::uint32_t closeTime =
+                getApp().timeKeeper().closeTime().time_since_epoch().count();
             if (lCloseTime <= closeTime)
             {
                 std::uint32_t age = closeTime - lCloseTime;
                 if (age < 1000000)
                     l[jss::age] = Json::UInt (age);
             }
```

Comment:
Do you mind putting an:
else
    l[jss::age] = Json::UInt (0);

here? Otherwise, "age" doesn't get displayed at all. This is for the case where the close time is the next 10s interval that hasn't been reached yet. I think "0" is better than nothing at all, especially for monitoring tools.